
syntax = "proto3";

package dronecorerpc;

// Interface exported by the server.
service DroneCoreRPC {
}

message Empty {}
